packages/opds-web-client/src/stylesheets/root.scss

```diff
@@ -139,24 +164,29 @@ a {
 }
 
 // override bootstrap
-.navbar-default, .navbar-collapse {
+.navbar-default,
+.navbar-collapse {
   background-color: $dark;
   border: none;
-  color: white;
+  border-radius: 0;
+  color: $light;
+  height: 4.25rem;
```

Comment:
This height is causing some problems with the header dropdown in the admin interface when it's collapsed on small screens.